apps/ui/src/composables/useSpaceAlerts.ts

```diff
@@ -118,6 +124,20 @@ export function useSpaceAlerts(
     return expirationTime < now && expirationTime >= now - graceThresholdMs;
   });
 
+  const isRelayerBalanceLow = computed(() => {
+    if (isOffchainSpace.value) return false;
+    if (
+      !(
+        space.value.authenticators.includes(ETH_SIGNATURE_AUTHENTICATOR) &&
+        space.value.authenticators.length === 1
+      )
+    ) {
+      return false;
+    }
+
+    return !relayerInfo.value?.hasMinimumBalance;
```

Comment:
The alert message will appear even if the relayer balance is 0? This will be misleading, saying the gasless voting will be disabled while it's not enabled yet